include/amici/exception.h

```diff
@@ -39,79 +39,85 @@ namespace amici {
             va_end(ap);
             storeBacktrace(12);
         }
-        
+
         AmiException(const AmiException& old) {
             /** copy constructor
              * @param old object to copy from
              */
             snprintf(msg, sizeof(msg), "%s", old.msg);
             snprintf(trace, sizeof(trace), "%s", old.trace);
         }
-        
+
         /** override of default error message function
          * @return msg error message
          */
         const char* what() const throw() {
             return msg;
         }
-        
+
         /** returns the stored backtrace
          * @return trace backtrace
          */
         const char *getBacktrace() const {
             return trace;
         }
-        
+
         /** stores the current backtrace
          * @param nMaxFrames number of frams to go back in stacktrace
          */
         void storeBacktrace(const int nMaxFrames) {
             std::ostringstream trace_buf;
-            
+
             #ifdef PLATFORM_WINDOWS
-            
+
             trace_buf << "stacktrace not available on windows platforms\n";
-            
+
             #else
-            
+
             void *callstack[nMaxFrames];
             char buf[1024];
             int nFrames = backtrace(callstack, nMaxFrames);
             char **symbols = backtrace_symbols(callstack, nFrames);
-            
-            for (int i = 2; i < nFrames; i++) { // start at 2 to omit AmiException and storeBacktrace call
+
+            for (int i = 2; i < nFrames;
+                 i++) { // start at 2 to omit AmiException and storeBacktrace
+                        // call
                 Dl_info info;
                 if (dladdr(callstack[i], &info) && info.dli_sname) {
                     char *demangled = NULL;
                     int status = -1;
                     if (info.dli_sname[0] == '_')
-                        demangled = abi::__cxa_demangle(info.dli_sname, NULL, 0, &status);
-                    snprintf(buf, sizeof(buf), "%-3d %*p %s + %zd\n",
-                             i-2, int(2 + sizeof(void*) * 2), callstack[i],
-                             status == 0 ? demangled :
-                             info.dli_sname == 0 ? symbols[i] : info.dli_sname,
-                             (char *)callstack[i] - (char *)info.dli_saddr);
+                        demangled = abi::__cxa_demangle(info.dli_sname, NULL, 0,
```

Comment:
Maybe we can at some point define a `clang-format` style for coherent formatting. Started in `.clang-format` (#168), but never really used.

Reply:
this was formatted by clang format :D

Reply:
Ah, I thought the old version was. Alright then ;).